xls/dslx/bytecode/bytecode_interpreter.cc

```diff
@@ -1503,7 +1503,7 @@ absl::Status ProcConfigBytecodeInterpreter::EvalSpawn(
       std::unique_ptr<BytecodeInterpreter> next_interpreter,
       CreateUnique(import_data, next_bf.get(), full_next_args, options));
   proc_instances->push_back(ProcInstance{proc, std::move(next_interpreter),
-                                         std::move(next_bf), full_next_args});
+                                         std::move(next_bf), full_next_args, type_info});
```

Comment:
nit: can you fix this to be `<= 80 columns wide`?

Reply:
Fixed @proppy